website/docs/guides/migration/versions/02-upgrading-to-v1.5.md

```diff
@@ -48,13 +48,16 @@ The following env vars have been renamed, for consistency with the convention fo
 
 As described in [dbt-core#7169](https://github.com/dbt-labs/dbt-core/pull/7169), command-line parameters that could be silent before will no longer be silent. See [dbt-labs/dbt-core#7158](https://github.com/dbt-labs/dbt-core/issues/7158) and [dbt-labs/dbt-core#6800](https://github.com/dbt-labs/dbt-core/issues/6800) for more examples of the behavior we are fixing.
 
+Finally: The [built-in `generate_alias_name` macro](https://github.com/dbt-labs/dbt-core/blob/1.5.latest/core/dbt/include/global_project/macros/get_custom_name/get_custom_alias.sql) has been updated to handle versioned models. In order to start defining versioned models, projects which have reimplemented the `generate_alias_name` macro with custom logic will need to update those custom implementations. This is not a requirement before upgrading to v1.5, only for using the new feature.
```

Comment:
Indeed, it's not a _strict_ requirement before upgrading.

But it feels like a ticking time bomb if they don't update their customized `generate_alias_name` as part of their upgrade. 💣 

i.e., if they upgrade now but don't try to start using model versions until 6+ months from now, they could really be scratching their heads trying to figure out why things are not working as documented. 💥